com.github.gkarsay.parlatype.json

```diff
@@ -2,30 +2,24 @@
     "app-id": "com.github.gkarsay.parlatype",
 
     "runtime": "org.gnome.Platform",
-    "runtime-version": "3.32",
+    "runtime-version": "3.34",
     "sdk": "org.gnome.Sdk",
 
     "command": "parlatype",
 
     "finish-args": [
         "--share=ipc",
-        "--socket=x11",
```

Comment:
First of all, thank you! I'm not sure about removal of `--socket=x11`. Other commits, like this one https://gitlab.gnome.org/GNOME/gnome-maps/commit/6af17720d1dc528eb8fc29b9dac76fa21511e217 just add a fallback-x11 socket without removing the x11 socket, suggesting that it is still needed on "older" flatpak versions. Whatever "older" is. Do you know more about this?

Reply:
Yes, you can read more about it in the flatpak docs 

> The fallback-x11 option makes the X11 socket available only if there is no Wayland socket. This option was introduced in 0.11.3. To support older Flatpak releases, specify both x11 and fallback-x11. The fallback-x11 option takes precedence when both are supported.

Reply:
Thanks, 0.11.3 is good enough. On my distro, Debian, this means it will work on oldstable (Stretch) with backports enabled.